src/core_plugins/kibana/public/dashboard/dashboard_container_api.js

```diff
@@ -1,26 +0,0 @@
-import { ContainerAPI } from 'ui/embeddable';
```

Comment:
Instead of communicating via this API and using a slew of functions, this has been replaced via communication with state only, and two functions, one for each communication direction.